parsers/CA_SK.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+from logging import Logger, getLogger
+from typing import List, Optional
+
+from pytz import timezone
+from requests import Session
+
+from parsers.lib.exceptions import ParserException
+
+TIMEZONE = timezone("America/Regina")
+
+PRODUCTION_URL = (
+    "https://www.saskpower.com/ignitionapi/PowerUseDashboard/GetPowerUseDashboardData"
+)
+
+PRODUCTION_MAPPING = {
+    "Hydro": "hydro",
+    "Wind": "wind",
+    "Solar": "solar",
+    "Natural Gas": "gas",
+    "Coal": "coal",
+    "Other": "unknown",  # This is internal consumption and losses.
```

Comment:
If you look at the details for "Other" on https://www.saskpower.com/Our-Power-Future/Our-Electricity/Electrical-System/Where-Your-Power-Comes-From, it seems to also include Heat Recovery Facilities from pipeline compressor stations and some "Small Independent Power Producers". It seems fine to map it to `unknown`, but probably include them in the comment